string class_id
---
uint32[] instance_ids
geometry_msgs/PoseStamped[] poses